packages/expo-linking/src/createURL.ts

```diff
@@ -58,16 +58,19 @@ function ensureLeadingSlash(input: string, shouldAppend: boolean): string {
  * Helper method for constructing a deep link into your app, given an optional path and set of query
  * parameters. Creates a URI scheme with two slashes by default.
  *
- * The scheme in bare and standalone must be defined in the Expo config (`app.config.js` or `app.json`)
- * under `expo.scheme`.
+ * The scheme must be defined in the Expo config (`app.config.js` or `app.json`) under `expo.scheme`.
  *
  * # Examples
  * - Bare: `<scheme>://path` - uses provided scheme or scheme from Expo config `scheme`.
  * - Standalone, Custom: `yourscheme://path`
```

Comment:
Some suggestions: clarify precedence rules (always helpful in docs!), combine bare and standalone and custom into just "Development and production builds". Reading the code, I did not see any difference between them. [resolveScheme()](https://github.com/expo/expo/blob/7204c425c62306b7b72c01e1e065a52f97905377/packages/expo-linking/src/Schemes.ts#L96-L193) mainly checks for `ExecutionEnvironment.StoreClient` (aka Expo Go).

```suggestion
 * The scheme must be defined in the Expo config (`app.config.js` or `app.json`) under `expo.scheme` or `expo.{android,ios}.scheme`. Platform-specific schemes defined under `expo.{android,ios}.scheme` take precedence over universal schemes defined under `expo.scheme`.
 *
 * # Examples
 * - Development and production builds: `<scheme>://path` - uses the optional `scheme` property if provided, and otherwise uses the first scheme defined by your Expo config
```